tox.ini

```diff
@@ -17,7 +17,7 @@ deps=
     -r{toxinidir}/requirements/test.txt
 
     2.7: -r{toxinidir}/requirements/test-ci-default.txt
-    3.4,3.5,3.6: -r{toxinidir}/requirements/test-ci-default.txt
+    3.4,3.5,3.6,3.7: -r{toxinidir}/requirements/test-ci-default.txt
```

Comment:
Could this line be merged with the one above it?

Reply:
I believe it could be but I assumed that there was some method behind the was it was.